osx_run.sh

```diff
@@ -1,53 +1,7 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -euf -o pipefail
 set -a
 
-export NODE_ENV=development
-export DEBUG=True
-export PORT=8079
-export COVERAGE_DIR=htmlcov
-export DATABASE_URL=postgres://postgres:postgres@localhost:5432/postgres
-export MICROMASTERS_USE_WEBPACK_DEV_SERVER=True
-export MICROMASTERS_SECURE_SSL_REDIRECT=False
-export MICROMASTERS_DB_DISABLE_SSL=True
+. ./_osx_setup.sh
```

Comment:
can you explain more about the purpose of _run.sh and _start.sh? naming could probably be better, since semantically they would seem to be doing the same thing (eg: "i want to start my machine" and "i want to run my machine" mean roughly the same thing). this file just seems to be running setup then executing the user's arguments as a command. is that going to be useful to us? also, what are the use cases where a user would want to execute _start?